x-pack/solutions/observability/plugins/synthetics/public/apps/synthetics/hooks/use_monitor_detail_locator.test.ts

```diff
@@ -1,90 +0,0 @@
-/*
```

Comment:
@fkanout Since you kept the logic for timeRange and tabId, I am wondering why you removed this test file. I would bring it back.

cc @dominiqueclarke